cmd/docs.go

```diff
@@ -37,18 +39,33 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.CliConfiguration{}, err)
 			}
 
+			// Detect terminal width if not specified in `atmos.yaml`
+			maxWidth := cliConfig.Settings.Docs.MaxWidth
+			if maxWidth == 0 {
+				if term.IsTerminal(int(os.Stdout.Fd())) {
+					w, _, err := term.GetSize(int(os.Stdout.Fd()))
```

Comment:
In this context `w` is overly vague. I think it would be more clear if we called it screenWidth and separetly define the desired width as the `min(screenWidth, maxWidth, 120)`

```suggestion
					screenWidth, _, err := term.GetSize(int(os.Stdout.Fd()))
```

Reply:
Just refactored the logic to keep `screenWidth` at 120 unless the conditions specify otherwise, making it more predictable and flexible